internal/operators/manager_test.go

```diff
@@ -231,6 +274,13 @@ var _ = Describe("Operators manager", func() {
 				&models.ClusterOperator{OperatorType: models.OperatorTypeLso, Enabled: swag.Bool(true)},
 			},
 		),
+		table.Entry("when only CNV is specified",
+			models.Operators{&models.ClusterOperator{OperatorType: models.OperatorTypeCnv, Enabled: swag.Bool(true)}},
+			models.Operators{
+				&models.ClusterOperator{OperatorType: models.OperatorTypeCnv, Enabled: swag.Bool(true)},
+				&models.ClusterOperator{OperatorType: models.OperatorTypeLso, Enabled: swag.Bool(true)},
```

Comment:
That should probably be `OperatorTypeOcs` and `OperatorTypeLso`.

Reply:
`OperatorTypeLso` is fine